src/lightning/app/launcher/launcher.py

```diff
@@ -0,0 +1,480 @@
+import inspect
+import logging
+import os
+import signal
+import sys
+import time
+import traceback
+from functools import partial
+from multiprocessing import Process
+from typing import Callable, Dict, List, Optional, Tuple, TypedDict
+
+from lightning_utilities.core.imports import module_available
+
+ENABLE_MULTIPLE_WORKS_IN_DEFAULT_CONTAINER = bool(int(os.getenv("ENABLE_MULTIPLE_WORKS_IN_DEFAULT_CONTAINER", "0")))
+
+
+if module_available("lightning_app"):
+    from lightning_app import LightningFlow
+    from lightning_app.core import constants
+    from lightning_app.core.api import start_server
+    from lightning_app.core.queues import MultiProcessQueue, QueuingSystem
+    from lightning_app.storage.orchestrator import StorageOrchestrator
+    from lightning_app.utilities.cloud import _sigterm_flow_handler
+    from lightning_app.utilities.component import _set_flow_context, _set_frontend_context
+    from lightning_app.utilities.enum import AppStage
+    from lightning_app.utilities.exceptions import ExitAppException
+    from lightning_app.utilities.load_app import extract_metadata_from_app, load_app_from_file
+    from lightning_app.utilities.proxies import WorkRunner
+    from lightning_app.utilities.redis import check_if_redis_running
+
+    try:
+        from lightning_app.utilities.app_commands import run_app_commands
+
+        ABLE_TO_RUN_APP_COMMANDS = True
+    except (ImportError, ModuleNotFoundError):
+        ABLE_TO_RUN_APP_COMMANDS = False
+
+else:
+    from lightning.app import LightningFlow
+    from lightning.app.core import constants
+    from lightning.app.core.api import start_server
+    from lightning.app.core.queues import MultiProcessQueue, QueuingSystem
+    from lightning.app.storage.orchestrator import StorageOrchestrator
+    from lightning.app.utilities.cloud import _sigterm_flow_handler
+    from lightning.app.utilities.component import _set_flow_context, _set_frontend_context
+    from lightning.app.utilities.enum import AppStage
+    from lightning.app.utilities.exceptions import ExitAppException
+    from lightning.app.utilities.load_app import extract_metadata_from_app, load_app_from_file
+    from lightning.app.utilities.proxies import WorkRunner
+    from lightning.app.utilities.redis import check_if_redis_running
+
+    try:
+        from lightning.app.utilities.app_commands import run_app_commands
+
+        ABLE_TO_RUN_APP_COMMANDS = True
+    except (ImportError, ModuleNotFoundError):
+        ABLE_TO_RUN_APP_COMMANDS = False
+
+
+if ENABLE_MULTIPLE_WORKS_IN_DEFAULT_CONTAINER:
+    from lightning.app.launcher.lightning_hybrid_backend import CloudHybridBackend as CloudBackend
+else:
+    from lightning.app.launcher.lightning_backend import CloudBackend
+
+if True:
```

Comment:
is this just for debugging?

Reply:
This is to make qa happy with non-top imports